components/tools/OmeroWeb/omeroweb/filesessionstore.py

```diff
@@ -184,7 +195,7 @@ def save(self, must_create=False):
                     os.unlink(output_file_name)
 
         except (OSError, IOError, EOFError):
-            pass
+            logger.debug(traceback.format_exc())
```

Comment:
The `exc_info` attribute is always available to the logger so there is no real need to import `traceback` and call `format_exc()` here.